lms/envs/common.py

```diff
@@ -2080,6 +2080,9 @@
 
     # Email marketing integration
     'email_marketing',
+
+    # Eric's doing some profiling!
+    'django_extensions',
```

Comment:
This was needed to allow profile data to be gathered. With this `INSTALLED_APP`, the following command will run the lms and profile each request as it's made.

`python manage.py lms --settings=devstack runprofileserver --traceback --pythonpath=. --use-cprofile --prof-path=/tmp/profiling --contracts 0.0.0.0:8000`